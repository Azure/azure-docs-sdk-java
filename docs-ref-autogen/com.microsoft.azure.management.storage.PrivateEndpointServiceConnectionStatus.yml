### YamlMime:ManagedReference
items:
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  id: PrivateEndpointServiceConnectionStatus
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage
  children:
  - com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED
  - com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PENDING
  - com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()
  - com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.REJECTED
  - com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)
  - com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.values()
  langs:
  - java
  name: PrivateEndpointServiceConnectionStatus
  nameWithType: PrivateEndpointServiceConnectionStatus
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  type: Class
  package: com.microsoft.azure.management.storage
  summary: Defines values for PrivateEndpointServiceConnectionStatus.
  syntax:
    content: public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>
  inheritance:
  - java.lang.Object
  - com.microsoft.rest.ExpandableStringEnum
  inheritedMembers:
  - com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)
  - com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)
  - com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)
  - com.microsoft.rest.ExpandableStringEnum.hashCode()
  - com.microsoft.rest.ExpandableStringEnum.toString()
  - com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)
  - java.lang.Object.clone()
  - java.lang.Object.finalize()
  - java.lang.Object.getClass()
  - java.lang.Object.notify()
  - java.lang.Object.notifyAll()
  - java.lang.Object.wait()
  - java.lang.Object.wait(long)
  - java.lang.Object.wait(long,int)
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED
  id: APPROVED
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  langs:
  - java
  name: APPROVED
  nameWithType: PrivateEndpointServiceConnectionStatus.APPROVED
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED
  type: Field
  package: com.microsoft.azure.management.storage
  summary: Static value Approved for PrivateEndpointServiceConnectionStatus.
  syntax:
    content: public static final PrivateEndpointServiceConnectionStatus APPROVED
    return:
      type: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PENDING
  id: PENDING
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  langs:
  - java
  name: PENDING
  nameWithType: PrivateEndpointServiceConnectionStatus.PENDING
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PENDING
  type: Field
  package: com.microsoft.azure.management.storage
  summary: Static value Pending for PrivateEndpointServiceConnectionStatus.
  syntax:
    content: public static final PrivateEndpointServiceConnectionStatus PENDING
    return:
      type: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()
  id: PrivateEndpointServiceConnectionStatus()
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  langs:
  - java
  name: PrivateEndpointServiceConnectionStatus()
  nameWithType: PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()
  overload: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus*
  type: Constructor
  package: com.microsoft.azure.management.storage
  syntax:
    content: public PrivateEndpointServiceConnectionStatus()
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.REJECTED
  id: REJECTED
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  langs:
  - java
  name: REJECTED
  nameWithType: PrivateEndpointServiceConnectionStatus.REJECTED
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.REJECTED
  type: Field
  package: com.microsoft.azure.management.storage
  summary: Static value Rejected for PrivateEndpointServiceConnectionStatus.
  syntax:
    content: public static final PrivateEndpointServiceConnectionStatus REJECTED
    return:
      type: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)
  id: fromString(java.lang.String)
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  langs:
  - java
  name: fromString(String name)
  nameWithType: PrivateEndpointServiceConnectionStatus.fromString(String name)
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.fromString(String name)
  overload: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.fromString*
  type: Method
  package: com.microsoft.azure.management.storage
  summary: Creates or finds a PrivateEndpointServiceConnectionStatus from its string representation.
  syntax:
    content: public static PrivateEndpointServiceConnectionStatus fromString(String name)
    parameters:
    - id: name
      type: java.lang.String
      description: a name to look for
    return:
      type: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
      description: the corresponding PrivateEndpointServiceConnectionStatus
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.values()
  id: values()
  artifact: com.microsoft.azure:azure-mgmt-storage:1.36.1
  parent: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  langs:
  - java
  name: values()
  nameWithType: PrivateEndpointServiceConnectionStatus.values()
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.values()
  overload: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.values*
  type: Method
  package: com.microsoft.azure.management.storage
  syntax:
    content: public static Collection<PrivateEndpointServiceConnectionStatus> values()
    return:
      type: java.util.Collection<com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus>
      description: known PrivateEndpointServiceConnectionStatus values
references:
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus*
  name: PrivateEndpointServiceConnectionStatus
  nameWithType: PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus
  package: com.microsoft.azure.management.storage
- uid: java.lang.String
  spec.java:
  - uid: java.lang.String
    name: String
    fullName: java.lang.String
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.fromString*
  name: fromString
  nameWithType: PrivateEndpointServiceConnectionStatus.fromString
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.fromString
  package: com.microsoft.azure.management.storage
- uid: java.util.Collection<com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus>
  spec.java:
  - uid: java.util.Collection
    name: Collection
    fullName: java.util.Collection
  - name: <
    fullName: <
  - uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
    name: PrivateEndpointServiceConnectionStatus
    fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus
  - name: '>'
    fullName: '>'
- uid: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.values*
  name: values
  nameWithType: PrivateEndpointServiceConnectionStatus.values
  fullName: com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.values
  package: com.microsoft.azure.management.storage
- uid: com.microsoft.rest.ExpandableStringEnum<com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus>
  name: ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>
  nameWithType: ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>
  fullName: com.microsoft.rest.ExpandableStringEnum<com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus>
- uid: java.lang.Object.notify()
  name: Object.notify()
  nameWithType: Object.notify()
  fullName: java.lang.Object.notify()
- uid: java.lang.Object.wait()
  name: Object.wait()
  nameWithType: Object.wait()
  fullName: java.lang.Object.wait()
- uid: java.lang.Object.finalize()
  name: Object.finalize()
  nameWithType: Object.finalize()
  fullName: java.lang.Object.finalize()
- uid: java.lang.Object.notifyAll()
  name: Object.notifyAll()
  nameWithType: Object.notifyAll()
  fullName: java.lang.Object.notifyAll()
- uid: java.lang.Object.clone()
  name: Object.clone()
  nameWithType: Object.clone()
  fullName: java.lang.Object.clone()
- uid: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)
  name: ExpandableStringEnum.withNameValue(String,T,Class<T>)
  nameWithType: ExpandableStringEnum.withNameValue(String,T,Class<T>)
  fullName: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)
- uid: com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)
  name: ExpandableStringEnum.<T>fromString(String,Class<T>)
  nameWithType: ExpandableStringEnum.<T>fromString(String,Class<T>)
  fullName: com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)
- uid: com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)
  name: ExpandableStringEnum.<T>values(Class<T>)
  nameWithType: ExpandableStringEnum.<T>values(Class<T>)
  fullName: com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)
- uid: com.microsoft.rest.ExpandableStringEnum.toString()
  name: ExpandableStringEnum.toString()
  nameWithType: ExpandableStringEnum.toString()
  fullName: com.microsoft.rest.ExpandableStringEnum.toString()
- uid: com.microsoft.rest.ExpandableStringEnum.hashCode()
  name: ExpandableStringEnum.hashCode()
  nameWithType: ExpandableStringEnum.hashCode()
  fullName: com.microsoft.rest.ExpandableStringEnum.hashCode()
- uid: java.lang.Object.getClass()
  name: Object.getClass()
  nameWithType: Object.getClass()
  fullName: java.lang.Object.getClass()
- uid: java.lang.Object.wait(long)
  name: Object.wait(long)
  nameWithType: Object.wait(long)
  fullName: java.lang.Object.wait(long)
- uid: com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)
  name: ExpandableStringEnum.equals(Object)
  nameWithType: ExpandableStringEnum.equals(Object)
  fullName: com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)
- uid: java.lang.Object.wait(long,int)
  name: Object.wait(long,int)
  nameWithType: Object.wait(long,int)
  fullName: java.lang.Object.wait(long,int)
- uid: java.util.Collection
  name: Collection
  nameWithType: Collection
  fullName: java.util.Collection
- uid: com.microsoft.rest.ExpandableStringEnum
  name: ExpandableStringEnum
  nameWithType: ExpandableStringEnum
  fullName: com.microsoft.rest.ExpandableStringEnum
- uid: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class
  name: ExpandableStringEnum.withNameValue(String,T,Class
  nameWithType: ExpandableStringEnum.withNameValue(String,T,Class
  fullName: com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class
- uid: T>)
  name: T>)
  nameWithType: T>)
  fullName: T>)
- uid: com.microsoft.rest.ExpandableStringEnum.
  name: ExpandableStringEnum.
  nameWithType: ExpandableStringEnum.
  fullName: com.microsoft.rest.ExpandableStringEnum.
- uid: T>fromString(java.lang.String,java.lang.Class
  name: T>fromString(String,Class
  nameWithType: T>fromString(String,Class
  fullName: T>fromString(java.lang.String,java.lang.Class
- uid: T>values(java.lang.Class
  name: T>values(Class
  nameWithType: T>values(Class
  fullName: T>values(java.lang.Class
