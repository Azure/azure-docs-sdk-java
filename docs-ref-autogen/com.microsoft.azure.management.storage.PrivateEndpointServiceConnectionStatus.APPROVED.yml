### YamlMime:JavaMember
uid: "com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.microsoft.azure.management.storage.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.microsoft.azure.management.storage"
artifact: com.microsoft.azure:azure-mgmt-storage:1.38.1
